### ISSUE TITLE
centering text syntax inconsistency

### ISSUE THREAD

**Issue by jsnkuhn** [gameboy9/randoTracker#53]
We're using 'hAlign' for text alignment in the freetext tags but 'align' for alignment in the player tags. Probably should go with hAlign for both assuming we'll add vAlign at some point as well.

**Comment by Datyedyeguy** [comment]
Agreed.  There's some cleanup and such that I want to do, but I didn't want to shake things up right before the marathon.
